addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -34,6 +34,26 @@
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="INPUT_MAP" type="text" required="false">
```

Comment:
I'm a litte disappointed here, that this is not happening in a more automated fashion. But it doesn't hurt if we give the user the choice to do his own mapping, as long as we also do auto-detection, too.

Reply:
Do not be disappointed :) There is an actual auto detection happening by analyzing the AVR descriptor and other featues, see DeviceInformationXML, DeviceDescriptorXML, InputConverterXML.

This hidden setting, is an extra layer that will give user ability to remap and fine tune input names for his AVR. That said, this will be the last resort and I think the autodetection worked well on all the tested models.